File: problemas/1000/1012.js
const [a, b, c] = require('fs').readFileSync('/dev/stdin', 'utf-8').split(' ').map(x => Number(x));

console.log(`TRIANGULO: ${(a * c / 2.0).toFixed(3)}`);
console.log(`CIRCULO: ${(3.14159 * c * c).toFixed(3)}`);
console.log(`TRAPEZIO: ${((a + b) * c / 2.0).toFixed(3)}`);
console.log(`QUADRADO: ${(b * b).toFixed(3)}`);
console.log(`RETANGULO: ${(a * b).toFixed(3)}`);

File: problemas/1000/1012.dart
import 'dart:io';

void main() {
  final entrada = stdin.readLineSync().split(' ');
  final a = double.parse(entrada[0]);
  final b = double.parse(entrada[1]);
  final c = double.parse(entrada[2]);
  stdout.writeln('TRIANGULO: ${(a * c / 2).toStringAsFixed(3)}');
  stdout.writeln('CIRCULO: ${(3.14159 * c * c).toStringAsFixed(3)}');
  stdout.writeln('TRAPEZIO: ${((a + b) * c / 2).toStringAsFixed(3)}');
  stdout.writeln('QUADRADO: ${(b * b).toStringAsFixed(3)}');
  stdout.writeln('RETANGULO: ${(a * b).toStringAsFixed(3)}');
}